sympy/printing/tests/test_latex.py

```diff
@@ -60,8 +60,8 @@ class lowergamma(sym.lowergamma):
     pass   # testing notation inheritance by a subclass with same name
 
 
-x, y, z, t, w, a, b, c, s, p = symbols('x y z t w a b c s p')
-k, m, n = symbols('k m n', integer=True)
+x, y, z, t, w, a, b, c, s, p, x1, x2, n1, n2, d, k, f, i, j, e = symbols('x y z t w a b c s p x1 x2 n1 n2 d k f i j e')
+k, m, n, f, g, h = symbols('k m n f g h', integer=True)
```

Comment:
I don't think g and h are being used anywhere apart from `test_latex_decimal_separator`, where they are being redefined anyways. Also, f should not be declared as an integer (L2415), and you are defining it just above anyways.